src/components/DateTime.tsx

```diff
@@ -24,12 +24,12 @@ const DateTime: React.FC = () => {
         let updateMonth = 'Error';
         let updateDate = 'Error';
         let updateYear = 'Error';
-        const regexSearch = /([0-9]+)([0-9]+)([0-9]+)/;
+        const regexSearch = /([0-9]+)\/([0-9]+)\/([0-9]+)/;
         const searchResult = originalDate.match(regexSearch);
         if (searchResult != null) {
-            updateMonth = searchResult[2];
-            updateDate = searchResult[1];
-            updateYear = searchResult[0];
+            updateMonth = searchResult[1];
```

Comment:
Unfortunately these changes break the date 
![image](https://user-images.githubusercontent.com/14501811/110218870-4cd37080-7e79-11eb-8a26-600d427f26bd.png)